src/tools/compiletest/src/runtest.rs

```diff
@@ -3258,6 +3266,45 @@ impl<'test> TestCx<'test> {
         let stamp = crate::stamp(&self.config, self.testpaths, self.revision);
         fs::write(&stamp, compute_stamp_hash(&self.config)).unwrap();
     }
+
+    fn run_yk_tir_test(&self) {
+        let proc_res = self.compile_test();
+
+        if !proc_res.status.success() {
+            self.fatal_proc_rec("compilation failed!", &proc_res);
+        }
+
+        if !proc_res.status.success() {
```

Comment:
This is the same condition as the `if` above which seems odd.

Reply:
Oops. This is a copy/paste mistake. Nice catch.